web app/src/main/java/de/tinf15b4/kino/web/ui/SmartCinemaUi.java

```diff
@@ -5,6 +5,12 @@
 
 import javax.annotation.PostConstruct;
 
+import de.tinf15b4.kino.web.views.AccountView;
+import de.tinf15b4.kino.web.views.CinemaListView;
+import de.tinf15b4.kino.web.views.FavoriteListView;
+import de.tinf15b4.kino.web.views.LoginView;
+import de.tinf15b4.kino.web.views.MovieListView;
+import de.tinf15b4.kino.web.views.SearchResultsView;
```

Comment:
These import statements appear to have been improperly moved by your autoformatter. We want them in Eclipse ordering.